### PR TITLE
add UnicodeCompact and Clickhouse styles.

### PR DESCRIPTION
**Pull request by thda**
Hi,

this fixes incorrect column counts whit multiple result set and adds compact unicode and clickhouse styles.

Can you please let me know what you think of this ?

Regards

### CODE REVIEW

File: encode.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"io"
 
-	"github.com/mattn/go-runewidth"
+	runewidth "github.com/mattn/go-runewidth"
```
Comment (kenshaw) [inline encode.go]: Please do not change this. The problem with the package name, and the inclusion of the alias is that the URL path doesn't match the path on disk and the declared package name in the actual `go-runewidth` code. Additionally, this is a cosmetic change not related to the rest of this PR.

Comment (kenshaw) [reply encode.go]: To clarify the above, "... the rationale for the inclusion of the package alias is that ... "

File: util.go [hunk 2]
```diff
@@ -82,11 +82,13 @@ func DefaultTableSummary() map[int]func(io.Writer, int) (int, error) {
...
-	Mid  [4]rune
-	Row  [4]rune
-	Wrap [4]rune
-	End  [4]rune
+	Top          [4]rune
+	Mid          [4]rune
+	Row          [4]rune
+	Wrap         [4]rune
+	End          [4]rune
+	EmbedColumns bool
```
Comment (kenshaw) [inline util.go]: Please rename `EmbedColumns` to `Inline` and add a comment indicating what it is for. Specifically:

```go
~     // Inline causes the column header names to be rendered "inline" with the
+     // top table line.
+     Inline bool
```

File: util.go [hunk 3]
```diff
@@ -82,11 +82,13 @@ func DefaultTableSummary() map[int]func(io.Writer, int) (int, error) {
...
+	Compact      bool
```
Comment (kenshaw) [inline util.go]: I don't believe this option is necessary. This should be the same as borders = 0 and the `Wrap` line's spacer character as `0`.

**Comment by thda** [comment]
Rebasing and reworking

**Pull request closed without merging**